src/platforms/javascript/common/session-replay/troubleshooting.mdx

```diff
@@ -29,6 +29,8 @@ There's currently no support for `canvas`. It's being tracked in this [GitHub is
 
 The replay 'video' is actually a video-like reproduction of the HTML on your website. This means that all the external resources your site uses (CSS/Images/Fonts), will be rendered by the corresponding &lt;style&gt;, &lt;img&gt; and &lt;video&gt; tags on your site. Add `sentry.io` to your CORS policy so the iframe hosted on sentry.io can fetch and display these resources.
 
+Please note that only static, publicly hosted videos can be captured by Replay, e.g. `src="./my-video.mp4"`. Streamed videos and similar are not supported.
```

Comment:
```suggestion
Note that only static, publicly hosted videos (for example, `src="./my-video.mp4"`) can be captured by Replay. Streamed videos and similar are not supported.
```
We generally avoid latin acronyms (e.g.,) in docs. 

"and similar" is pretty vague, could this be clarified a bit more? If not, it's ok as is too.

Reply:
Not 100% sure about all the other cases how you can show a video 🤔 I'll merge this for now I think (with your suggestions, thank you!) and we can potentially tweak it later if we know of other specific scenarios!